tests/shell/shell_recorder/README.md

```diff
@@ -24,7 +24,9 @@ If File is preset but empty a fresh database temp file will be provided for ever
 
 Posix-extended regex is used to check and validate return values and outputs.
 
-Options: 
+**Remark:** Shell Recorder uses the `⏎` symbol as line terminator. This means that you need to use the character `⏎`  (instead of `\n`) if you want to match a line ending in a multiline output.
```

Comment:
> Can you describe at which positions this line terminator is accepted…

Basically we just use `⏎` instead of the `\n` symbol in a regex. This means `⏎` matches both a newline and `⏎` symbol. Just like any other normal character (e.g. `a`, `→`, `1` or `😱`) you can use `⏎` everywhere in the regex.

> …how to enter the character, …

This depends on how you insert “special characters” in your current OS. If someone does not know how to do that, one easy way is just to copy the character from the documentation or the test file `db_changes.esr` and then insert it via pasting.

> …and make an example how to use it?

I added an example and more documentation in the latest commits.